cmd/zb/delete.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	ispec "github.com/opencontainers/image-spec/specs-go/v1"
+	"gopkg.in/resty.v1"
+	"zotregistry.io/zot/errors"
+)
+
+func deleteUploadedFiles(manifestHash map[string]string, configHash map[string]string,
```

Comment:
Why are we not deleting the entire repo after the test? Wouldn't that be quicker/cleaner?